torch/_dynamo/guards.py

```diff
@@ -33,12 +34,17 @@
     tuple_iterator_len,
 )
 
+
 log = logging.getLogger(__name__)
 TensorGuards = torch._C._dynamo.guards.TensorGuards
 check_obj_id = torch._C._dynamo.guards.check_obj_id
 check_type_id = torch._C._dynamo.guards.check_type_id
 
 
+def tensor_identity_match(a, b):
+    return a is b
```

Comment:
It seems like the rest of these helper functions live in utils, this one should too.

Reply:
Also, it is not clear to me that you want to do this via closure vars. You can just generate the "is" directly in the Python code to be compiled into the guard, and that will be speedier (one opcode, versus a function call opcode)

Reply:
agreed